modules/downloading-manifest-files-resources-oci.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="downloading-manifest-files-resources-oci_{context}"]
+= Downloading manifest files and deployment resources
+
+You must download the archive file that includes files for creating cluster resources and custom manifests. After you extract the contents of the archive file, you must upload the extracted files to an {oci-first} stack. A stack is an {oci} feature where you can automate the provisioning all the necessary {oci} infrastructure resources, such as the custom image, that are required for installing an {product-title} cluster on {oci}.
+
+.Prerequisites
+
+* You uploaded a generated Agent ISO image to {oci}. See the "Using the Assisted Installer to generate an OCI-compatible Agent ISO image" section.
+* You have permissions to access the `oracle-quickstart / oci-openshift` GitHub repository.
+
+.Procedure
+
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, click the **<> Code** button and then click **Download ZIP**.  The following list details these resources:
++
+** link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests[CCM and CSI custom manifests].
+** link:https://github.com/oracle-quickstart/oci-openshift/blob/main/README.md[Download the {oci} Resource Manager configuration to provision resources for deploying Openshift on {oci}].
++
+[NOTE]
+====
+Consider using the example configurations in link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests/manifests[manifests] GitHub page as a guide for configuring a cloud infrastructure to meet your organization's needs.
+====
+
+. Go to **Resource Manager > Stacks** and click the **Create stack** button, and upload the archive file, `.zip`, that you downloaded previously from the `oracle-quickstart / oci-openshift` repository.  
+
+. Complete the following actions to create a stack:
++
+.. From the **Stack information** section, specify a name for your stack.
```

Comment:
I'm not sure but AFAIK we add CCM and CSI custom manifest to AI SaaS before start installation. 
And no need create Stack object 
@adriengentil Please, correct me

Reply:
the stack is to apply the terraform template provided by Oracle, it will create the resources in the cloud (instances, network, LB, ...)

Reply:
Should be "Developer services ->  Resource Manager > Stacks"

Reply:
I updated the parent step:

> Create a stack by completing the link:https://docs.oracle.com/en-us/iaas/Content/ResourceManager/Tasks/create-stack-compartment.htm[Creating a Stack from an Existing Compartment] procedure in the Oracle documentation. Ensure that you also complete the following subtasks:

Where subtasks list uploading the archive file and so on.

Reply:
@adriengentil Do we need customize downloaded files in oci-openshift-main.zip ?

Reply:
we need to customize some of the files after the stack and the resources are created: https://github.com/openshift/openshift-docs/pull/69901/files#diff-cc590dcc8b47227ade63367c532a4721928b1cd89273e8b8eaba2331411ef548R57-R59

Reply:
@adriengentil That's clear. But I asked before we uploaded to Stack.